pkg/virtual/workspaces/registry/rest.go

```diff
@@ -441,12 +441,12 @@ func (w withProjection) ResultChan() <-chan watch.Event {
 			}
 			w.ch <- ev
 		}
+		close(w.ch)
```

Comment:
would prefer this to be a defer'ed close

Reply:
Oh, OK. It was initially ;-)

Reply:
fixed